crates/shim/src/mount.rs

```diff
@@ -723,4 +723,47 @@ mod tests {
             assert_eq!(options, expected_options);
         }
     }
+
+    #[test]
+    fn test_mount_rootfs() {
+        let fs_type = Some("overlay");
+        let source = Some("overlay");
+        let workdir =
+            "/var/lib/containerd-test/io.containerd.snapshotter.v1.overlayfs/snapshots/0/work";
```

Comment:
To be honest, I don't think it's good idea to create test folder like this. Please use `/tmp/$RANDOM-LONG-STRING`. Thanks.